system/greenhouse-ccloud/templates/thanos-global.yaml

```diff
@@ -0,0 +1,76 @@
+{{- if and .Values.thanos.enabled .Values.thanos.globalClusters -}}
+kind: Secret
+apiVersion: v1
+metadata:
+  name: thanos-{{ .Chart.Name }}-tls
+  namespace: {{ .Chart.Name }}
+type: Opaque
+data:
+  ca.crt: {{ required ".Values.thanos.tls.caCert missing" .Values.thanos.tls.caCert | b64enc | quote }} 
+  tls.crt: {{ required ".Values.thanos.tls.tlsCert missing" .Values.thanos.tls.tlsCert | b64enc | quote }} 
+  tls.key: {{ required ".Values.thanos.tls.tlsKey missing" .Values.thanos.tls.tlsKey | b64enc | quote }}
```

Comment:
should we add this to plugin templates ?

Reply:
We need to store secret data in a Secret. In the plugin templates it will be exposed.

Reply:
template of the secret resource would be in the plugin, we would still populate values from secret no ?

Reply:
It's referenced right here: https://github.com/sapcc/helm-charts/blob/ea604ad96eaf3c36020ac53dbb886362ae3086c7/system/greenhouse-ccloud/templates/thanos-global.yaml#L57-L71

Reply:
it's all there already actually
https://github.com/cloudoperators/greenhouse-extensions/blob/main/thanos/charts/templates/secret-tls.yaml

Reply:
This is the secret deployed to the remote cluster so the Thanos query pod can mount it. The secret in this PR is deployed to the central cluster so multiple plugins can use the certificates.